spec/rails_helper.rb

```diff
@@ -0,0 +1,66 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
```

Comment:
fyi:
There is a good practice to use `DatabaseCleaner`
[source](https://github.com/DatabaseCleaner/database_cleaner)
This way you will not produce more and more records in test database